src/handlers/db/generalHandler.js

```diff
@@ -1,52 +1,37 @@
-const logger = require('../utils/logger');
-const models = require('../models/index');
-
-const tables = {
-  users: models.users,
-  admins: models.admins,
-};
+const logger = require('../../utils/logger');
+const db = require('../../database/index');
 
 const findAllEntries = (tableName) => {
   logger.debug('Getting all entries.');
-  return tables[tableName].findAll({});
+  return db.select().from(tableName);
 };
 
 const findEntryWithId = (tableName, id) => {
   logger.info(`Searching for entry ${id}`);
-  return tables[tableName].find({
-    where: {
-      id: id,
-    },
-  });
+  return db(tableName).where('id', id);
 };
 
 const findWithUsernameAndPassword = (tableName, username, password) => {
   logger.info(`Querying database for entry with username "${username}" and password "${password}"`);
-  return tables[tableName].findAll({
-    where: {
-      userName: username,
-      password: password,
-    },
+  return db(tableName).where({
+    userName: username,
+    password: password,
   });
 };
 
 const createNewEntry = (tableName, entry) => {
   logger.info('Creating entry');
-  return tables[tableName].create(entry);
+  return db(tableName).insert(entry).returning('*');
 };
 
-function updateEntry(entry, newEntry) {
```

Comment:
We should decide whether to use arrow functions or functions, but not mix between the two.